wallet/src/main/scala/org/bitcoins/wallet/models/AddressTagDb.scala

```diff
@@ -0,0 +1,28 @@
+package org.bitcoins.wallet.models
+
+import org.bitcoins.core.protocol.BitcoinAddress
+import org.bitcoins.core.wallet.utxo.{
+  AddressTag,
+  AddressTagName,
+  AddressTagType,
+  InternalAddressTag
+}
+
+case class AddressTagDb(
+    address: BitcoinAddress,
+    tagName: AddressTagName,
+    tagType: AddressTagType) {
+
+  val addressTag: AddressTag = InternalAddressTag(tagName, tagType)
+
+  def ==(at: AddressTagDb): Boolean =
```

Comment:
Why don't we test for equality with `tagType`?

Reply:
We do, we test equality of `addressTag` which uses both `tagName` and `tagType`